lms/djangoapps/commerce/utils.py

```diff
@@ -44,6 +44,13 @@ class EcommerceService(object):
     def __init__(self):
         self.config = CommerceConfiguration.current()
 
+    def toggle_commerce_config(self, enabled=False, checkout_page='/test_basket'):
```

Comment:
@rlucioni I added this method to allow me to turn on / off the config model from the test.  I was having issues importing the model directly in the test as the namespace was conflicting.